bin/nearley-test.js

```diff
@@ -28,6 +28,11 @@ var opts = nomnom
         abbr: 'o',
         help: "File to output to (defaults to stdout)",
     })
+    .option('quiet', {
+        abbr: 'q',
+        flag: true,
+        help: "Output parse table only omitting parsing steps",
```

Comment:
We should change this helptext. The parse table *is* the thing you call "parsing steps". Try `"Output parse results only (hide Earley table)"`.